backend/reservations/config.py

```diff
@@ -18,8 +18,8 @@ def env_config(val, default=''):
 LOGLEVEL = env_config('loglevel', 'info').upper()
 
 URL_HOST = env_config('host', 'localhost')
-URL_PORT = env_config('port', '3000')
-URL_SCHEMA = env_config('schema', 'http')
+URL_PORT = env_config('url_port', '3000')
+URL_SCHEMA = env_config('url_schema', 'http')
```

Comment:
This prolly wants to be https ja? I assume it will forward to https but not sure what layer that is happening on.

Reply:
defaults to http for dev envs. http will forward, and the staging/prod config  have it set as https.